core/src/identity/ed25519.rs

```diff
@@ -43,8 +43,9 @@ impl Keypair {
         self.0.to_bytes()
     }
 
-    /// Decode a keypair from the format produced by `encode`,
-    /// zeroing the input on success.
+    /// Decode a keypair from the [binary format](https://datatracker.ietf.org/doc/html/rfc8032#section-5.1.5)
+    /// produced by `encode`, zeroing the input on success.
```

Comment:
```suggestion
    /// produced by [`Keypair::encode`], zeroing the input on success.
```

while you're at it :-)